src/Orchard.Web/Modules/Orchard.Projections/Models/LayoutRecord.cs

```diff
@@ -9,6 +9,7 @@ public class LayoutRecord {
         }
 
         public virtual int Id { get; set; }
+        public virtual string GUIdentifier { get; set; }
```

Comment:
What is that?

Reply:
While implementing the setting, we noticed that an identifier for the query layouts was missing. The Guid is needed to correctly manage the import and export of the setting.